code/modules/clothing/under/accessories/accessory.dm

```diff
@@ -102,6 +102,9 @@
 	item_color = "stethoscope"
 
 /obj/item/clothing/accessory/stethoscope/attack(mob/living/carbon/human/M, mob/living/user)
+	if(istype(M, /mob/living/carbon/human/machine))
```

Comment:
You should use `M.get_species()` for the check. This way of doing things can break if I recall correctly.